central/telemetry/centralclient/interceptors_test.go

```diff
@@ -166,7 +166,7 @@ func Test_addCustomHeaders(t *testing.T) {
 			},
 		}
 		props := map[string]any{}
-		addCustomHeaders(rp, telemetryCampaign[1], props)
+		addCustomHeaders(rp, telemetryCampaign[2], props)
```

Comment:
Tests here are required to be run all. Running a single test causes panic :(

Reply:
#14660 fixes that.